ui/src/shared/components/AutoRefresh.js

```diff
@@ -9,7 +9,7 @@ const AutoRefresh = ComposedComponent => {
     constructor() {
       super()
       this.state = {
-        lastQuerySuccessful: false,
+        lastQuerySuccessful: true,
```

Comment:
What are the implications of this?

Reply:
This only affects when the component is initially rendered. Previously, since it was false, there would not be a loading sequence for when a graph was initially rendered. This change allows there to be one.